src/plugins/intel_gpu/src/graph/eye.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <eye_inst.h>
+#include <json_object.h>
+
+#include <sstream>
+
+#include "primitive_type_base.h"
+
+namespace cldnn {
+
+primitive_type_id eye::type_id() {
+    static primitive_type_base<eye> instance;
+    return &instance;
+}
+
+eye_inst::typed_primitive_inst(network& network, eye_node const& node) : parent(network, node) {}
+
+layout eye_inst::calc_output_layout(eye_node const& node) {
+    auto primitive = node.get_primitive();
+    return {*(primitive->output_data_type), cldnn::format::bfyx, primitive->output_shape};
```

Comment:
potentially we need to extract format from batch_shape here instead of direct bfyx format to be able to support blocked format

Reply:
let's postpone till Wednesday sync meeting